sesman.el

```diff
@@ -72,6 +73,21 @@ sessions running in dependent projects."
   :type 'boolean
   :package-version '(sesman . "0.3.2"))
 
+(defcustom sesman-follow-symlinks 'auto
+  "This variable determines whether symlinks should be followed.
+nil - Don't follow symlinks - use `expand-file-name' for expanding file paths.
+t - Follow symlinks - use `file-truename' for expanding file paths.
+'auto - Don't follow symlink unless it's under version control and
+`vc-follow-link' has nil value. Or `find-file-visit-truename' is non-nil."
+  :group 'sesman
+  :type '(choice (const :tag "Behave like `find-file'" auto)
+                 (const :tag "Don't follow symlinks"   nil)
+                 (const :tag "Follow symlinks"         t))
+  :package-version '(sesman . "0.3.2"))
+(put 'sesman-follow-symlinks
```

Comment:
@vspinu  White-listed `sesman-follow-symlinks`. 

Now I'm gone fore real. Good luck and thanks for the package :smile: